src/com/scottyplunkett/server/PageContent.java

```diff
@@ -0,0 +1,41 @@
+package com.scottyplunkett.server;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+
+class PageContent {
+    private String build;
+
+    PageContent(Path pagesPath, String requested) throws IOException {
+        build = build(pagesPath, requested);
```

Comment:
I'm seeing the same variable / method name pattern here.  Maybe you're looking for an internal variable name that has a clear connection to the operation creating it?  Some ideas:

* `read` / `read` = `requestContent` / `readRequestContent`
* `build` / `build` = `pageContent` / `buildPageContent`

Basically, `theValueName` / `theProcessName`.